code/modules/mob/living/silicon/robot/robot_hud.dm

```diff
@@ -25,14 +25,14 @@
 /mob/living/silicon/robot/check_HUDdatum()//correct a datum?
 	var/mob/living/silicon/robot/H = src
 
-	if (H.defaultHUD == "BorgStyle") //���� � ������� ���� �������� �����\��� ����
-		if(global.HUDdatums.Find(H.defaultHUD))//���� ���������� ����� ��� ����
+	if (H.defaultHUD == "BorgStyle") //���� � ������� ���� �������� �����\��� ����
```

Comment:
Oh god it happened again. @drexample do these need to be removed as well?

Reply:
This is actually not corrupted unciode but some strange version of russian.

Reply:
DESTROY THE CHIL- 
As long as it compiles, probably some github-byond-git unicode bullshit.
Yay for optimizations.